src/clients/FlexbaseClient.Patron.ts

```diff
@@ -16,4 +31,68 @@ export class FlexbaseClientPatron extends FlexbaseClientBase {
             return [];
         }
     }
+
+    async addPatron(patronData: IPatronData): Promise<IResponseUpdatePatron> {
```

Comment:
Why not use https://github.com/flexbase-eng/flexbase-client/blob/main/src/clients/FlexbaseClient.Patron.ts

Reply:
I don't do the add function there because if we don't pass the default `patronId` I see that it returns the array with all the patterns that exist and if it has `patronId` then it updates it. It would be necessary to pass another parameter to know if what is wanted is to add a new one.

I personally like to do more individual functions because if I call a function called `getPatrons` in the front end, I assume it is a query, but not a function to update.

Reply:
Because this is a `post`